src/contract_client/utils.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * The default timeout for waiting for a transaction to be included in a block.
  */
-export const DEFAULT_TIMEOUT = 10;
+export const DEFAULT_TIMEOUT = 60;
```

Comment:
this should be communicated **loudly** in the CLI release notes that will incorporate this, imo

Reply:
indeed, will add an entry